instruqt-tracks/consul-life-of-a-developer/track.yml

```diff
@@ -9,8 +9,9 @@ tags: []
 owner: hashicorp
 developers:
 - lance@hashicorp.com
+- hari@hashicorp.com
 private: true
 published: true
 show_timer: true
 skipping_enabled: true
-checksum: "9017204642515175710"
+checksum: "2032359557772946896"
```

Comment:
If you can add a commit to strip this checksum that will make this less likely to break CircleCI tests.

Reply:
@omkensey why would this break CircleCI test?

Reply:
See above comment -- the same applies here.